libfwupd/fwupd-device.c

```diff
@@ -659,11 +660,13 @@ fwupd_device_set_vendor (FwupdDevice *device, const gchar *vendor)
  * fwupd_device_get_vendor_id:
  * @device: A #FwupdDevice
  *
- * Gets the device vendor ID.
+ * Gets the combined device vendor ID.
  *
  * Returns: the device vendor, e.g. 'USB:0x1234', or %NULL if unset
```

Comment:
I think that this needs to be updated to show what a combined ID looks like

Reply:
I used the `|` char as a logical OR; other variables like this already use `,` -- any preference?

Reply:
seems fine to me.

Reply:
Example updated.